src/org/zaproxy/zap/extension/callback/ExtensionCallback.java

```diff
@@ -53,12 +54,18 @@
             .getLogger(ExtensionCallback.class);
 
     public ExtensionCallback() {
+    	super(NAME);
```

Comment:
The change of the name will break code that is relying on the "older" name (this was introduced in 2.6.0, if an add-on uses that name it will break in 2.7.0; not a problem for zap-extensions add-ons though, using the class instead of the name).